src/Microsoft.AspNet.JsonPatch/Exceptions/JsonPatchException.cs

```diff
@@ -3,14 +3,13 @@
 
 using System;
 using Microsoft.AspNet.JsonPatch.Operations;
-using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.JsonPatch.Exceptions
 {
-    public class JsonPatchException<TModel> : JsonPatchException where TModel : class
+    public class JsonPatchException : JsonPatchExceptionBase //where TModel : class
```

Comment:
Delete instead of commenting out.

Do we really need the base-class at all here?